app/controllers/tags_controller.rb

```diff
@@ -26,7 +26,6 @@ def show
       format.rss do
         expires_in 0, public: true
 
-        limit     = params[:limit].present? ? [params[:limit].to_i, PAGE_SIZE_MAX].min : PAGE_SIZE
```

Comment:
I am not confident about the handling of limit here

Reply:
Seems like the variable was not used anywhere. It should be used in the limit call on the line below.